### PR TITLE
Updated and pruned version of Sprints page from 2023

### PR DESCRIPTION
**Pull request by cybit**
Question: Will there a mentored sprint?

Apart from that is this a copy of  last years sprints page with  the sprints removed and  some stuff updated.

### CODE REVIEW

**Comment by hypha** [comment]
@cybit I added the image now.

File: src/content/pages/sprints.mdx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+facilities but the sprints are organised by **YOU**. It is a great chance to
+contribute to open-source projects large and small, learn from each other, geek
+out and have fun. 🐍
+
+Lunch and coffee will be provided.
+
+**When**: 13-14 July 2024 (09:30 - 17:00)
+
+**Where**: Rajska Building (RB),
+[Žižkov Campus, VŠE](https://www.vse.cz/english/about-vse/profile/campus/)
```
Comment (egeakman) [inline src/content/pages/sprints.mdx:27]: I think we should link google maps instead of the campus page

File: src/content/pages/sprints.mdx [hunk 2]
```diff
@@ -0,0 +1,126 @@
+---
+title: EuroPython 2024 Sprints
+subtitle: Organise or join a sprint in EuroPython 2024!
+social_card: sprints.png
+---
+
+import { Image } from "astro:assets";
+
+import sprintImage from "./images/sprint_collection_2420x1619.png";
```
Comment (egeakman) [inline src/content/pages/sprints.mdx:9]: Is any photo from last year's sprints included in this collection photo?

File: src/content/pages/sprints.mdx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  on the ground floor of RB
+
+{/*
+TODO: link map / page about sprint venue
+*/}
+
+**Who can attend**:
+
+- If you have an in-person EuroPython ticket
+  ([Conference, Tutorial or Combined](s#ticket-types)), then access to the
```
Comment (egeakman) [inline src/content/pages/sprints.mdx:42]: Shouldn't it be like this?

```suggestion
  ([Conference, Tutorial or Combined](/tickets#ticket-types)), then access to the
```

File: src/content/pages/sprints.mdx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  sprints is already included. Remember to select your sprints attendance when
+  buying / configuring your ticket, to let the organisers know.
+- If you do not have an in-person conference ticket, you can register to join
+  the sprints for free. Registration information will be announced soon
+  [here](tickets#sprint-tickets).
```
Comment (egeakman) [inline src/content/pages/sprints.mdx:47]: This section doesn't exist in the tickets page.

File: src/content/pages/sprints.mdx (hunk 1 above)
Comment (egeakman) [inline src/content/pages/sprints.mdx:27]:
```suggestion
[Žižkov Campus, VŠE](https://maps.app.goo.gl/azRTAczu8B5ma1XFA)
```

Comment (egeakman) [reply src/content/pages/sprints.mdx:27]: @cybit Can you check if this is the right place? Searching "Rajska Building (RB), Žižkov Campus, VŠE (Prague University of Economics and Business)" gives the wrong place:

image


The link I added is the right place I believe:

image

File: src/content/pages/sprints.mdx (hunk 2 above)
Comment (hypha) [reply src/content/pages/sprints.mdx:9]: Pretty sure not since I made this photo years ago :smile:

File: src/content/pages/sprints.mdx [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+You may want to come to the [Mentored Sprint](/mentored-sprint) on Friday before going to the actually sprint! Mentored Sprint is tailed to beginners who have not contributed to open-source before. We will have mentors who will be there to help you and give you a head start of your contributing journey.
+*/}
+
+## Organising a Sprint
+
+**Sprint rooms**: we will have 11 rooms in total for the sprints, all of which
+are on the 1st floor of the Rajska Building (RB) of VŠE. Check out the
+[room layout](https://drive.google.com/file/d/1N2Q_jB3j4LRXS15WJI2STlXvBkGABbgC/view?usp=sharing)
+for details. The rooms 103-107 and 111-116 each have the capacity for 24 and
```
Comment (hypha) [inline src/content/pages/sprints.mdx:71]: I don't think 111 is there. 
```suggestion
for details. The rooms 103-107 and 112-116 each have the capacity for 24 and
```

File: src/content/pages/sprints.mdx (hunk 2 above)
Comment (egeakman) [reply src/content/pages/sprints.mdx:9]: @hypha Opened a thread on Discord to make a new one!

**Review by hypha — COMMENTED** [review]
Left a comment with a minor thing.
But separately, I updated quite a few places for sprints, e.g. the where page for directions, as well as putting it on the front page. I am not sure if you want to add some of those in this PR as well or open another one later. But here is the reference: https://github.com/EuroPython/website/pull/253

File: src/content/pages/sprints.mdx (hunk 1 above)
Comment (cybit) [reply src/content/pages/sprints.mdx]: @egeakman Almost. The entrance is in between the two pictures. Where the blue square is.

Comment (egeakman) [reply src/content/pages/sprints.mdx]: @cybit Can you generate a new google maps link? I'm not sure where the exact entrance was, and don't trust my memory that much :D

Comment (hypha) [reply src/content/pages/sprints.mdx]: I just wanted to mention that Mia got quite a lot of details on how to get there last year. You can see it here: https://ep2023.europython.eu/where/#v%C5%A1e---sprints-venue.

Comment (egeakman) [reply src/content/pages/sprints.mdx]: > I just wanted to mention that Mia got quite a lot of details on how to get there last year. You can see it here: https://ep2023.europython.eu/where/#v%C5%A1e---sprints-venue.

Yes. It was my silliness at the time to not the check that page first :facepalm: I only checked the sprints page and that was all

Comment (egeakman) [reply src/content/pages/sprints.mdx]: Pointing out to ``/where/#v%C5%A1e---sprints-venue`` in the ``/sprints`` page might be a good idea though

Comment (hypha) [reply src/content/pages/sprints.mdx]: Absolutely! I think we just forgot about it last year!

**Comment by hypha** [comment]
@cybit do you want us to populate sprints related information in other pages, e.g. in Where, FAQ and homepage or you want to merge this as soon as possible? If you want the former, happy to try to commit to this existing PR if it saves some trouble.

**Review by rodrigogiraoserrao — COMMENTED** [review]
Just showing some pulse here, given that I'll be taking a look over sprints PRs.
(The info looks good to me.)

**Comment by egeakman** [comment]
@cybit can you add the image of your choice from the Discord thread as the new sprints collection photo?

File: src/content/pages/sprints.mdx (hunk 2 above)
Comment (cybit) [reply src/content/pages/sprints.mdx:9]: After your last comment only one file was left.

File: src/content/pages/sprints.mdx (hunk 4 above)
Comment (cybit) [reply src/content/pages/sprints.mdx]: I removed the sprints part, but left the tickets part.

File: src/content/pages/sprints.mdx (hunk 1 above)
Comment (cybit) [reply src/content/pages/sprints.mdx:34]: Tried to incorporate Mias text from last year.

**Comment by cybit** [comment]
@hypha , @egeakman I was finally able to work through you comments.

@hypha normally I would really like your references from  #253  here. But I think getting this one out now and then fixing later might be the better idea. 

@egeakman I would have preferred the hexagons but with your comment on the picture at the button left I used the other one.

**Comment by egeakman** [comment]
Let's remove the old picture ([sprint_collection_2420x1619.png](https://github.com/EuroPython/website/pull/693/files#diff-27a39fe8ea6e796b571b3d1798ab54f98d4dab53ed21f9763ac9a9d999f4a91e)) if we are not using it.

Other than that, all looks good. Thank you @cybit!

**Comment by hypha** [comment]
@cybit looks good to be shipped (after removing the old photo)! Thank you! Happy to work on adding references to other pages in another PR.

**Comment by cybit** [comment]
Photo is gone. I am merging.

**Pull request merged**